djangocms_versioning_filer/fields/file.py

```diff
@@ -143,9 +143,10 @@ def __init__(self, **kwargs):
     def formfield(self, **kwargs):
         # This is a fairly standard way to set up some defaults
         # while letting the caller override them.
+        remote_field = self.rel if hasattr(self, 'rel') else self.remote_field
```

Comment:
If a change is required for this PR can you add a quick comment here that dj 2.2 changed the field "rel".